### ISSUE TITLE
Cut-off park label: Hubbell Trading Post NHS

### ISSUE THREAD

**Issue by mtaylorlong** [nationalparkservice/park-tiles#164]
In Park Tiles 2.1 (all versions), the label for Hubbell Trading Post NHS is cut off almost completely at z7.

<img width="380" alt="screen shot 2015-08-19 at 2 39 21 pm" src="https://cloud.githubusercontent.com/assets/4741443/9368625/99d7619c-4681-11e5-97c0-922b52284895.png">

The point fall just within the tile, but since the name is long and ldir='W', it's running way off the tile (I think). Could we fix this by changing ldir to 'E', or would that create new problems at different zoom levels?

Once resolved, here's a checklist for where the fix should be made:
- [ ] Park Tiles 2.1 Fused (base and POI)
- [ ] Park Tiles 2.1 Satellite (base and POI)
- [ ] Park Tiles 2.1 Slate (base and POI)
- [ ] Park Tiles 2.1 Light (base and POI)

**Comment by makella** [comment]
I am pretty sure this was an oversight on my part when we were first building Park Tiles with TileMill 2. Several months after we had released, Mapnik got upgraded to version 3 in Studio and we weren't aware of it.  I wrote AJ about several labeling issues that arose with the upgrade and addressed many of them it looks like I didn't do the Hubbell Trading Post fix.

From AJ (emailed on 10/24/14):

Hey Mamata,

Here is a more complete list of changes: https://github.com/mapnik/mapnik/blob/master/CHANGELOG.md#300

Most of the changes should not affect existing styles, but some that might:

- Line-wrapping now works on labels placed along lines. This may affect existing projects where the same text styles are being used for both line labels and point labels.
- Kerning of characters is slightly different which may cause differences in collisions/visibility of some labels.
- Wrap-width is now correctly adjusted for scale_factor. Retina tiles may look different now but should be "more correct."
- Bugfix in shield rendering where the full area of the shield image was not being taken into account for collision testing. If you have styles that tried to work around this with "shield-min-padding" then the padding can be removed.  

__I think the second point about kerning may be what's causing the issue with Hubbell Trading Post.__ The new interpretation of the font is spacing it slightly wider which causes it to collide with Wupatki. A quick fix would be to manually tighten up the text with "text-character-spacing: -0.25;" or a similar small negative value.
